avocado/utils/kernel.py

```diff
@@ -97,14 +97,40 @@ def build(self):
         """
         log.info("Starting build the kernel")
         if self.config_path is None:
-            build.make(self.linux_dir, extra_args='O=%s defconfig' % self.build_dir)
+            build.make(self.linux_dir, extra_args='O=%s defconfig' %
+                       self.build_dir)
         else:
-            build.make(self.linux_dir, extra_args='O=%s olddefconfig' % self.build_dir)
+            build.make(self.linux_dir, extra_args='O=%s olddefconfig' %
+                       self.build_dir)
         build.make(self.linux_dir, extra_args='O=%s' % self.build_dir)
 
     def __del__(self):
         shutil.rmtree(self.work_dir)
 
+    def kernel_build_time(self, threads, timefile='/dev/null', make_opts='',
+                          output='/dev/null'):
```

Comment:
@PraveenPenguin How about adding this as an option for in build.make() method itself, https://github.com/avocado-framework/avocado/blob/master/avocado/utils/build.py#L21, that would make it more generic.
```

def make(path, make='make', env=None, extra_args='', ignore_status=False, allow_output_check='none', measure_time =False):
....
....
if measure_time:
       ......
        cmd = "time %s" % cmd
        .....
        .....

```

Some thing like this?